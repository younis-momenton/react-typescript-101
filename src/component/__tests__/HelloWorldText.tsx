import * as React from 'react';
import * as ReactDOM from 'react-dom';
import * as TestUtils from 'react-dom/test-utils';

const HelloWorld = require("../HelloWorld");

it("renders with crashing", () => {
    const helloWorld = TestUtils.renderIntoDocument(
        <HelloWorld name="Younis" />
    );

    const helloWorldNode = ReactDOM.findDOMNode(helloWorld);

    expect(helloWorldNode.textContent).toEqual("Hello World from Younis");
})