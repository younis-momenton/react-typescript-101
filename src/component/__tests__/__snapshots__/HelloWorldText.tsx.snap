// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component rendering compared to the snapshot 1`] = `
<div>
  <h1>
    Hello World from 
    test
  </h1>
</div>
`;
